blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -96,6 +96,17 @@ public String getStringProperty(String name) {
         return props.getProperty(name);
     }
 
+    public static String getRawStringProperty(Enum<? extends ConfigDefaults> name) {
+        return getRawStringProperty(name.toString());
+    }
+    public static String getRawStringProperty(String name) {
+        return props.getProperty(name);
+    }
+
+    public static boolean containsKey(String key) {
```

Comment:
@shintasmith I imagine so, but this class isn't really in a position to be mocked right now. Nevertheless, I'll change it.